src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,214 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class CarDaoImpl implements CarDao {
+
+    @Override
+    public Car create(Car car) {
+        String query = "INSERT INTO cars (model, manufacturer_id) VALUE (?, ?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection
+                        .prepareStatement(query, Statement.RETURN_GENERATED_KEYS)) {
+            statement.setString(1, car.getModel());
+            statement.setLong(2,car.getManufacturer().getId());
+            statement.executeUpdate();
+            ResultSet generatedKeys = statement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                Long id = generatedKeys.getObject(1, Long.class);
+                car.setId(id);
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException("Can't insert car into DB" + car, e);
+        }
+        insertDrivers(car);
+        return car;
+    }
+
+    @Override
+    public Optional<Car> get(Long id) {
+        String query = "SELECT c.id as car_id, model, manufacturer_id, manufacturers.name "
+                + "AS manufacturers_name, manufacturers.country AS manufacturers_country"
+                + " FROM cars c JOIN  manufacturers ON manufacturer_id = manufacturers.id "
```

Comment:
```suggestion
                + " FROM cars c JOIN  manufacturers m ON manufacturer_id = manufacturers.id "
```
you are using an alias for cars table, lets use it for manufacturers as well